app/javascript/dashboard/helper/specs/editorHelper.spec.js

```diff
@@ -6,6 +6,8 @@ import {
   cleanSignature,
   extractTextFromMarkdown,
   insertAtCursor,
+  findNodeToInsertImage,
```

Comment:
I moved the `findNodeToInsertImage` helper to the `editorHelper.js` file from the `messageEditorHelper.js` file. To use all the helpers in a single file. And deleted the `messageEditorHelper.js` file and its spec.